docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -206,6 +173,14 @@ address. To override the public address, specify the `public_addr` field:
   public_addr: "jira.example.com"
 ```
 
+The certificate for the public address must be installed in the proxy.
+
+<Admonition type="Note" title="Subdomains Only for Teleport Cloud">
```

Comment:
We can scope this Admonition for Cloud users and remove "for Teleport Cloud" from the title. A cleaner alternative might be to wrap the whole "Customize public address" H3 section in a `ScopedBlock` so Cloud users don't see it.

Reply:
@ptgott put in scopedblock

Reply:
👍 In that case, I don't think we need this `Admonition`, since a user with the `cloud` scope won't see it.